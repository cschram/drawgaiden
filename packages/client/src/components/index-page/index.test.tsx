import React from 'react';
import renderer from 'react-test-renderer';
import { IndexPage } from './index';
import testData from '../../__testdata__/canvas';

test('<App/>', () => {
    const props = {
        lastCanvasID: '',
        createCanvas: () => {},
        joinCanvas: (id: string) => {}
    };
    const tree = renderer.create(<IndexPage {...props}/>).toJSON();
    expect(tree).toMatchSnapshot();
});