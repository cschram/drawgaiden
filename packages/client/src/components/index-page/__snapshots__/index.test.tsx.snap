// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> 1`] = `
<div
  className="index-page"
>
  <div
    className="index-page__main"
  >
    <img
      alt="Draw Gaiden"
      src={Object {}}
    />
    <p>
      Create or rejoin a canvas to start drawing, and share the link with friends to collaborate.
    </p>
    <div>
      <button
        className="btn btn--primary"
        disabled={false}
        onClick={[Function]}
      >
        <span>
          Create Canvas
        </span>
      </button>
      <button
        className="btn btn--disabled"
        disabled={true}
        onClick={[Function]}
      >
        Rejoin Canvas
      </button>
    </div>
    <p>
      Having issues? Check our 
      <a
        href="https://github.com/drawgaiden/drawgaiden/issues"
        target="_blank"
      >
        bug tracker
      </a>
      .
    </p>
  </div>
</div>
`;
